registry/handlers/referrers.go

```diff
@@ -80,6 +97,19 @@ func (h *referrersHandler) GetReferrers(w http.ResponseWriter, r *http.Request)
 		referrers = []v1.Descriptor{}
 	}
 
+	// only consider pagination if # of referrers is greater than page size
+	if len(referrers) > pageSize {
```

Comment:
This `if` statement is redundant.

Reply:
deleted.

Reply:
Added it back after addressing the case `startIndex > len(referrers)`.